axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/AccountingBatchSupplyChainService.java

```diff
@@ -1,32 +0,0 @@
-/*
- * Axelor Business Solutions
- *
- * Copyright (C) 2023 Axelor (<http://axelor.com>).
- *
- * This program is free software: you can redistribute it and/or  modify
- * it under the terms of the GNU Affero General Public License, version 3,
- * as published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package com.axelor.apps.supplychain.service;
-
-import com.axelor.apps.account.db.AccountingBatch;
-import com.axelor.apps.bankpayment.service.batch.AccountingBatchBankPaymentService;
-import com.axelor.apps.base.db.Batch;
-import com.axelor.apps.supplychain.service.batch.BatchAccountingCutOffSupplyChain;
-import com.axelor.inject.Beans;
-
-public class AccountingBatchSupplyChainService extends AccountingBatchBankPaymentService {
-
-  @Override
-  public Batch accountingCutOff(AccountingBatch accountingBatch) {
-    return Beans.get(BatchAccountingCutOffSupplyChain.class).run(accountingBatch);
```

Comment:
You cannot just simply remove this without adding anything, we need this override, but maybe you need to find another way to do it

Reply:
No need to find another way to do it, it's already done since BatchAccountingCutOffSupplyChain extends and is binded to BatchAccountingCutOff, when we call the .run on BatchAccountingCutOff it'll use the one from BatchAccountingCutOffSupplyChain which override it.